cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/azure-mc-common/AzureClusterSettingsStep.tsx

```diff
@@ -127,6 +152,16 @@ export function AzureClusterSettingsStep(props: Partial<StepProps>) {
                             selectedInstanceId={selectedInstanceTypeId}
                         />
                     </div>
+                    <div cds-layout="col:12">
+                        <RetrieveOSImages
```

Comment:
Sorry I think I missed this in your previous PR for the component name. I would rename the component to OsImageSelect

Reply:
Done